_posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown

```diff
@@ -0,0 +1,120 @@
+
+To improve the UI / UX of the OpenSearch Playground, one of the primary ways that new users can evaluate the OpenSearch platform, we tried to view our product through the fresh eyes of a new customer. To do so we deployed the research principles of a UX heuristics evaluation. In this process, we identified four key recommendations. These recommendations are informed by the most common violations of tenets and traps and UX heuristics that our researchers identified in the Playground and are summarized below.
+
+![](/assets/media/blog-images/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product/Image.jpg)
+**Focus on workflows that help customers achieve a goal rather than complete specific tasks**. The Playground home page offers customers concrete tasks like ingesting data, exploring data, and trying a query assistant. Think about organizing the site to support customer goals such as proactive anomaly monitoring, visualizing your data, and getting started with minimal expertise.
+
+**Use consistent terminology and workflows across all tools and features to help customers work faster and build intuitive mental models.** There are several instances where the experience changes depending on whether users are logged in or not. There are also instances where terms like “buckets” are used that differ from industry standards. Simplifying terminology and workflows to follow industry knowledge and best practices will increase customer confidence that this product is for them.
+
+**Minimize available actions within the Playground that trial users cannot access**. A recurring Tenet & Trap violation was Inviting Dead End. Users encounter options (e.g. “add sample data”) within the Playground that they do not have access to and are unable to perform. Focusing on eliminating these encounters and errors make it easier for potential customers to assess the platform.
+
+**Investigate potential content gaps on [OpenSearch.org](http://opensearch.org/) which** may hinder new user trial. The customer journey on [OpenSearch.org](http://opensearch.org/) seems to have information gaps that add friction to understanding and trialing of the Playground.
+
+
+## Takeaways
+
+* Perform a heuristics analysis to rapidly identify common traps and violations in your product design
+* You can scope your research as broadly or narrowly as you need
+* UX heuristics insights are often universally applicable to your entire UX; be sure to broaden your understanding (and dissemination) of your findings to ensure maximum value from your efforts
+* You can engage outside UX research experts or utilize existing resources to apply these UI Tenets and Traps and 10 Usability Heuristics to your product
+* Implement a regular Heuristics research practice to monitor and track improvements.
+
+## References
```

Comment:
Deleted this section, as we already link to and explain these resources earlier in the blog post.